docs/configuration.rst

```diff
@@ -14,22 +14,149 @@ interpret a rawstring (``r''`` or ``r""``), whereas double quotes
 python (``""`` or ``''``). This matters for paths on Windows, for which
 we advice to use single quotes.
 
-.. code:: toml
+.. literalinclude:: config/metamod.toml
+    :language: toml
 
-   # This is a configuration file for the imod_coupler
-   # Relative paths are interpreted as relative to the configuration file path
+Config schema
```

Comment:
Could you add for each variable what it roughly does? I think for most users names like ``log_level`` and ``mf6_msw_recharge_pkg`` are not self-explanatory. For example, the explanation for ``mf6_msw_recharge_pkg``  could be: "_Package name (specified in the Modflow 6 .nam file) of the recharge package to which MetaSWAP will be coupled_".